posthog/hogql_queries/web_analytics/external_clicks.py

```diff
@@ -43,36 +43,38 @@ def to_query(self) -> ast.SelectQuery:
                 """
 SELECT
     url AS "context.columns.url",
-    uniq(filtered_person_id) AS "context.columns.visitors",
-    sum(filtered_click_count) AS "context.columns.clicks"
+    tuple(uniqIf(filtered_person_id, {current_period}), uniqIf(filtered_person_id, {previous_period})) AS "context.columns.visitors",
+    tuple(sumIf(filtered_click_count, {current_period}), sumIf(filtered_click_count, {previous_period})) AS "context.columns.clicks"
 FROM (
     SELECT
         any(person_id) AS filtered_person_id,
         count() AS filtered_click_count,
-        {url_expr} AS url
+        {url_expr} AS url,
+        MIN(timestamp) AS start_timestamp
```

Comment:
This will be the timestamp of the first click rather than the start of the session. If you want the start of the session you can do `any(session.$start_timestamp) as start_timestamp`

Reply:
good catch